client/components/ProductOverview/ThumbnailView.jsx

```diff
@@ -19,6 +19,9 @@ const ThumbnailView = ({ photos, handleClick, currentIndex }) => {
       } else if (currentIndex < 7) {
         setThumbnailStart(0);
         setThumbnailEnd(7);
+      } else {
+        setThumbnailStart(currentIndex);
+        setThumbnailEnd(currentIndex + 7);
```

Comment:
Without knowing what were doing, it's hard to know what this 7 represents. A nice variable like `THUMBNAIL_UPPER_LIMIT` might help to clear things up.

Reply:
Made 7 into a constant.